mobile/src/main/res/layout/media_griditem.xml

```diff
@@ -34,45 +36,47 @@
         android:layout_centerInParent="true"
         android:paddingLeft="@dimen/activity_horizontal_margin"
         android:paddingRight="@dimen/activity_horizontal_margin"
-        android:scaleType="centerInside"
         android:alpha="0.4"
+        android:contentDescription="@null"
+        android:scaleType="centerInside"
         android:src="@drawable/butter_logo" />
 
     <ImageView
         android:id="@+id/cover_image"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
+        android:contentDescription="@null"
```

Comment:
we should handle accessibility correctly at some point. I'll create a separate ticket for that.